slam-libraries/viam-orb-slam3/orbslam_server_v1.h

```diff
@@ -39,6 +41,19 @@ class SLAMServiceImpl final : public SLAMService::Service {
     ::grpc::Status GetMap(ServerContext *context, const GetMapRequest *request,
                           GetMapResponse *response) override;
 
+    /*
+      For a given GetPointCloudMapRequest
+      Returns a GetPointCloudMapResponse containing a sparse
+      slam map as a Binary PCD file
+
+      Map uses:
+      1. right hand rule
```

Comment:
[q] not sure if its worth stating that we use the right hand rule as the left hand rule is only ever used when looking at magnetic fields but clarity is never a bad thing

Reply:
done